scripts/babylon-integration/register-op-consumer-fp.sh

```diff
@@ -32,6 +32,9 @@ echo
 sleep 5
 
 echo "Registering OP consumer finality provider..."
-docker exec consumer-finality-provider /bin/sh \
-    -c "/bin/fpd register-finality-provider $OP_FP_EOTS_PK_HEX"
+OP_FP_REGISTRATION_TX_OUTPUT=$(docker exec consumer-finality-provider /bin/sh \
+    -c "/bin/fpd register-finality-provider $OP_FP_EOTS_PK_HEX")
+echo "OP consumer finality provider registration tx output: $OP_FP_REGISTRATION_TX_OUTPUT"
```

Comment:
there is no need to have this line. Line 34 already has the prompt

Reply:
it is different, if the tx is failed, we can get the failed msg.

Reply:
rigth but my point is we can change this line to
```
echo $OP_FP_REGISTRATION_TX_OUTPUT"
```
since we already have

```
echo "Registering OP consumer finality provider..."
```